.travis/before_install.sh

```diff
@@ -64,18 +64,9 @@ cd /tmp || die
     }
 
     [ $BUILD_TARGET != posix-app-pty ] || {
-        sudo apt-get install socat expect libdbus-1-dev autoconf-archive || die
+        sudo apt-get install socat expect || die
         JOBS=$(getconf _NPROCESSORS_ONLN)
         (
-        WPANTUND_TMPDIR=/tmp/wpantund
```

Comment:
Just double-checking. This change here is not related to the main fix, right?

Reply:
no, not related to the main fix, just by the way.

Reply:
Thanks.